src/common/options/global.yaml.in

```diff
@@ -1880,6 +1880,15 @@ options:
   services:
   - mon
   with_legacy: true
+- name: mon_config_history_size
+  type: size
+  level: advanced
+  desc: Defines the number of config history entries to be kept in monitor store 
```

Comment:
Suggest

desc: Defines the number of config history entries to be kept in the Monitors' store